services/spar/src/Spar/Scim/User.hs

```diff
@@ -336,9 +318,9 @@ createValidScimUser tokinfo vsu@(ST.ValidScimUser uref handl mbName richInfo act
 
   -- FUTUREWORK(arianvp): these two actions we probably want to make transactional
   lift . wrapMonadClient $ Data.writeScimUserTimes storedUser
-  lift . wrapMonadClient $ Data.insertSAMLUser uref buid
+  (lift . wrapMonadClient . (`Data.insertSAMLUser` buid)) `mapM_` uref
```

Comment:
Could we use `traverse_` or `for_` instead of `mapM_` ?

Reply:
```suggestion
  for_ uref $ (lift . wrapMonadClient . (`Data.insertSAMLUser` buid))
```